doc/userManual.md

```diff
@@ -195,7 +195,13 @@ If not, there should be something wrong in the **TcpServer**.
 
 The `TcpServer` module uses the `GenRead` user event to send the explanation of the errors that occur internally.
 The error is cleared by the module and the user has to manage it.
-The error explanation comes from the error LabVIEW internal database.
+
+The **TcpServerBase.clearErrorAndSendExplanation.vi** sends the generic user event string with the error code and the source of the error.
```

Comment:
Please help to put the link of NI error code list that the user knows where to look for the error details:
https://www.ni.com/docs/en-US/bundle/labview/page/lverror/misc_lv_error_codes.html

Maybe there are more. Please help to double check.

Reply:
Done.